### ISSUE TITLE
Help: Where are offline Nativescript Angular Typescript firebase offline examples?

### ISSUE THREAD

**Issue by mikepaggi** [EddyVerbruggen/nativescript-plugin-firebase#469]
It would be great to see some examples of using offline mode from Firebase for listviews in NativeScript Angular using Typescript.  Any help here?  I saw the giftler example but it doesn't do offline since the way they implemented it.

**Comment by jlooper** [comment]
@mikepaggi I need to update Giftler...but I've been working actively on QuickNoms (https://github.com/jlooper/quicknoms-app) which does support offline viewing of recipes (the images stored in Firebase don't seem to come through, but the text does) - just by setting the persist: true flag on init. I haven't done more than this and it's not a listview but maybe this may help:
```
firebase.init({
  persist: true
```

**Comment by Dashue** [comment]
Yeah I thought it would require some advance setup and searched the internet like crazy only to find I just requires setting that outfit to true. AFAIK keeps up to 10mb cached and can be overridden with keepinsync settings on value change listeners

**Comment by jlooper** [comment]
I guess you can close this issue?
